### PR TITLE
packet-cli: 0.5.0 -> metal-cli 0.6.0

### PR DESCRIPTION
**Pull request by teutat3s**
###### Motivation for this change
packet-cli is now metal-cli 
https://github.com/equinix/metal-cli/releases/tag/v0.6.0

###### Things done
- renamed packet-cli to metal-cli
- bumped metal-cli version to v0.6.0

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by nshalman — APPROVED** [review]
LGTM

**Review by Br1ght0ne — APPROVED** [review]
LGTM.

**Comment by colemickens** [comment]
Can someone help me understand -> This doesn't build?

```
❯ nix build github:teutat3s/nixpkgs/36587c22b381af8b65a71d3355537e245fb9651e#legacyPackages.x86_64-linux.metal-cli
error: builder for '/nix/store/3669zf5dszz1x897k2sy00d7jy8dail7-metal-cli-0.6.0.drv' failed with exit code 1;
       last 10 log lines:
       > Building subPackage ./internal/projects
       > Building subPackage ./internal/ssh
       > Building subPackage ./internal/twofa
       > Building subPackage ./internal/users
       > Building subPackage ./internal/vlan
       > Building subPackage ./tests
       > github.com/equinix/metal-cli/tests
       > go build github.com/equinix/metal-cli/tests: no non-test Go files in /build/source/tests
       > installing
       > ln: failed to create symbolic link '/nix/store/d0vvm75yfj60x7vjvwb5xgzc28dnsv3h-metal-cli-0.6.0/bin/metal': File exists
       For full logs, run 'nix log /nix/store/3669zf5dszz1x897k2sy00d7jy8dail7-metal-cli-0.6.0.drv'.

~ 27s

```

**Comment by teutat3s** [comment]
@colemickens It does build, but it doesn't install, because the final binary seems to be already there. Are you sure the command `metal` isn't already available?

File: pkgs/development/tools/metal-cli/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    sha256 = "sha256-dGeOFrsqhW0+aQyB4f6pvv4ZBawqKX2+WRskDWoLS7E=";
+  };
+
+  vendorSha256 = "sha256-ifSfeJjrZI1Hrsq64zAGBiLVc8GKvq+Ddg26gQooyTs=";
+
+  postInstall = ''
+    ln -s $out/bin/metal-cli $out/bin/metal
+  '';
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/development/tools/metal-cli/default.nix:20]: Please always add comments why tests are disabled.

**Pull request merged**